ui/src/app/pages/profile/data-access-panel.tsx

```diff
@@ -1,82 +1,72 @@
-import {StyledAnchorTag} from 'app/components/buttons';
-import {CheckCircle, ControlledTierBadge} from 'app/components/icons';
+import * as React from 'react';
```

Comment:
This new version does a good deal less than the previous, which is an unlaunched implementation of a previous version of the mocks.  See https://precisionmedicineinitiative.atlassian.net/browse/RW-6256 and https://github.com/all-of-us/workbench/pull/4803 for the history.

We'll probably add equivalent functionality back in for https://precisionmedicineinitiative.atlassian.net/browse/RW-7059 once we've finalized the appearance/behavior.

Reply:
I do not have access to the new mocks but I believe you that this matches them 🤷